model/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+
+plugins {
+    java
+}
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(15))
```

Comment:
We don't want the copy-paste like this one in each module.

Have you seen how it's done in `kanban`? Please check out [its build script](https://github.com/spine-examples/kanban/blob/master/build.gradle.kts) (and the script-plugins it references, they are in its `buildSrc/kotlin` folder).